From c857238b09d2e53f957e55b012d42d8bb25b2a8d Mon Sep 17 00:00:00 2001
From: Daniel Baluta <daniel.baluta@nxp.com>
Date: Mon, 15 Oct 2018 20:09:00 +0300
Subject: [PATCH 4864/5242] MLK-19949: ASoC: dsp: Fix codec libraries search
 path

commit  33f3c39a407d38e0405aa3b8fb77724cf566ead0 from
https://source.codeaurora.org/external/imx/linux-imx.git

Old implementation uses /usr/lib/imx-mm/audio-codec
to look for codec libraries.

Also this is the patch where Yocto rootfs stores the codec
libraries. So until we align Yocto with the kernel lets
change the driver to use the old path.

Signed-off-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/fsl_dsp_xaf_api.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/fsl/fsl_dsp_xaf_api.c b/sound/soc/fsl/fsl_dsp_xaf_api.c
index 503503a..f40560b0 100644
--- a/sound/soc/fsl/fsl_dsp_xaf_api.c
+++ b/sound/soc/fsl/fsl_dsp_xaf_api.c
@@ -183,8 +183,8 @@ int xaf_comp_create(struct xf_client *client, struct xf_proxy *proxy,
 
 	memset((void *)p_comp, 0, sizeof(struct xaf_comp));
 
-	strcpy(lib_path, "/lib/firmware/imx/dsp/");
-	strcpy(lib_wrap_path, "/lib/firmware/imx/dsp/");
+	strcpy(lib_path, "/usr/lib/imx-mm/audio-codec/dsp/");
+	strcpy(lib_wrap_path, "/usr/lib/imx-mm/audio-codec/dsp/");
 
 	p_handle = &p_comp->handle;
 
-- 
1.7.9.5

